src/app/shared/stache/modules/code-block/code-block.component.ts

```diff
@@ -4,6 +4,7 @@ declare let Prism: any;
 import 'prismjs/prism';
 import 'prismjs/components/prism-typescript';
 import 'prismjs/plugins/normalize-whitespace/prism-normalize-whitespace';
+require('style-loader!prismjs/themes/prism.css');
```

Comment:
What's the reasoning for adding it to every component, instead of the module, once?